.github/workflows/pytest.yaml

```diff
@@ -1,22 +1,9 @@
 name: pytest-workflow Check
 
 on:
-  workflow_run:
-    workflows: ["Workflows Docker Build"]
-    types:
-      - completed
+  workflow_call:
 
 jobs:
-  on-success:
-    runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
-    steps:
-      - run: echo 'The build workflow passed, starting tests'
-  on-failure:
-    runs-on: ubuntu-latest
-    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
-    steps:
-      - run: echo 'The build workflow failed, skipping tests' && exit 1
   list-tags:
```

Comment:
I don't think it's important, but I wonder if the tag list could be passed between the workflows. It's cheap to recompute, so it's more a curiosity.